### ISSUE TITLE
What is the advantages of drupal-project over drupal-composer-init or vanilla drupal > 8.3

### ISSUE THREAD

**Issue by gagarine** [drupal-composer/drupal-composer-project#434]
I just want one good and obvious way to setup Drupal.

Sadly their is tow documentation pages:
 https://www.drupal.org/docs/develop/using-composer/using-composer-to-manage-drupal-site-dependencies 
https://www.drupal.org/docs/user_guide/en/install-composer.html

And it's really not obvious witch way should be used.

**Comment by normanlolx** [comment]
As you already created this issue inside the repo of one of the mentioned possibilities you might as well just start with it and then find the answer to the question yourself. There is no "one good and obvious way". But the way this exact project here offers you, is probably the most common and most accepted nowadays. Good luck!

**Comment by rodrigoaguilera** [comment]
This is an issue that is being discussed at https://www.drupal.org/project/drupal/issues/2002304

**Comment by gagarine** [comment]
@leymannx I do test stuff, but I do prefer to know why I'm choosing somethings and what direction the community is taking if I start a fresh project. I started to try to cleanup the documentation on D.O., but realize it was such a mess and that I didn't have the information that I gave up.  For example the part "Option C: drupal/drupal (deprecated)" can be removed? On top of that, I don't have the right to edit the page on the user guide.

@rodrigoaguilera this issue was created in 2013.
